tests/Console/ConfigurationResolverTest.php

```diff
@@ -246,10 +246,10 @@ public function testResolvePathRelative()
     {
         $this->resolver
             ->setCwd(__DIR__)
-            ->setOption('path', 'Foo'.DIRECTORY_SEPARATOR.'Bar')
+            ->setOption('path', 'Command')
```

Comment:
Because now it will call `finder->in(...)`, which would blow up with exception of non-existing `Foo\Bar` dir.

before it was only setting the property (which was later used in `finder->in(...)` call and then throw exception, but utest didn't went so far)